src/client/pages/__eyedropper.vue

```diff
@@ -34,25 +41,46 @@ onMounted(() => {
     color.value = res.sRGBHex
   })
 })
+
+function ErrorBoundary() {
+  let content: JSX.Element = <div></div>
+  if (!inSecurityContext) {
+    content = <p>
+    EyeDropper is not available due to <a class="text-primary transition-colors hover:text-primary/80" href="https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts" target='_blank'>secure context</a> restrict.
```

Comment:
EyeDropper is available only in [secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) (HTTPS)